jerk/src/main/scala/org/ensime/jerk/JerkFormats.scala

```diff
@@ -30,6 +30,31 @@ private object JerkConversions extends DefaultJsonProtocol with FamilyFormats {
     def write(dtid: DebugThreadId): JsValue = JsNumber(dtid.id)
   }
 
+  implicit object EitherFormat extends JsonFormat[Either[L, R]] {
```

Comment:
oh, we don't need these... they should be provided by spray-json already. Although there is a bug in the compiler that means the objects need to be brought into scope so that they can be seen, like with the `SymbolFormat`.